client/src/components/RelatedProducts/Cards/CarouselR.jsx

```diff
@@ -1,18 +1,19 @@
-import { React, useState } from 'react';
+/* eslint-disable react/prop-types */
+/* eslint-disable max-len */
+/* eslint-disable no-shadow */
+import { React, useState, useEffect } from 'react';
 import styled from 'styled-components';
 import { FiChevronLeft, FiChevronRight } from 'react-icons/fi';
 import { Individualcard } from './Individualcard';
 
-function CarouselRelated({ zippedArray }) {
+function CarouselRelated({ informationArray }) {
```

Comment:
`informationArray` could be a bit more semantic. It's hard for me to figure out what that "means". What information is in this array? Just some observations about naming things.